api/src/test/java/org/openmrs/api/db/ProgramWorkflowDAOTest.java

```diff
@@ -9,19 +9,21 @@
  */
 package org.openmrs.api.db;
 
-import static org.junit.jupiter.api.Assertions.assertEquals;
-import static org.junit.jupiter.api.Assertions.assertNotNull;
-
-import java.util.Date;
-import java.util.List;
-
+import org.junit.Assert;
```

Comment:
Since we are migrating to JUnit 5, do you mind taking a look at this for the very small changes that you need to make? https://wiki.openmrs.org/display/docs/How+to+migrate+to+JUnit+5

Reply:
The ProgramWorkflowDAOTest is already running on JUnit 5 in the current master. So please rebase your changes on the current master. We put our static imports on top, can you please make sure you configure that in your IDE see https://wiki.openmrs.org/display/docs/How-To+Setup+And+Use+Your+IDE

Reply:
@dkayiwa @teleivo I've updated the tests to JUnit5. I've also configured my IDE and reformatted my code as per the style guides. Thanks.